src/component/mxgraph/shape/activity-shapes.ts

```diff
@@ -30,6 +31,24 @@ export abstract class BaseActivityShape extends mxRectangleShape {
     // enforced by BPMN
     this.isRounded = true;
   }
+
+  public paintForeground(c: mxAbstractCanvas2D, x: number, y: number, w: number, h: number): void {
+    super.paintForeground(c, x, y, w, h);
+    this.paintMarkerIcon(buildPaintParameter(c, x, y, w, h, this, 0.17, false, 1.5));
+  }
+
+  protected paintMarkerIcon(paintParameter: PaintParameter): void {
+    switch (StyleUtils.getBpmnMarker(this.style)) {
```

Comment:
Decision after discussion with @csouchet 
- we keep this PR as it is now
- 2 others PR will be created
  - a 1st one, soon to manage marker and collapsed 'icon' with the same way in the mxGraph style. This will prepare management of other markers, helps to manage markers position later
  - prior supporting a new marker (probably multi-instanciation), the model objects will be updated to store several markers